// bitwise/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.string = exports.nibble = exports.integer = exports.byte = exports.buffer = exports.bits = void 0;
var bits_1 = __importDefault(require("./bits"));
exports.bits = bits_1.default;
var buffer_1 = __importDefault(require("./buffer"));
exports.buffer = buffer_1.default;
var byte_1 = __importDefault(require("./byte"));
exports.byte = byte_1.default;
var integer_1 = __importDefault(require("./integer"));
exports.integer = integer_1.default;
var nibble_1 = __importDefault(require("./nibble"));
exports.nibble = nibble_1.default;
var string_1 = __importDefault(require("./string"));
exports.string = string_1.default;
var bitwise = { bits: bits_1.default, buffer: buffer_1.default, byte: byte_1.default, integer: integer_1.default, nibble: nibble_1.default, string: string_1.default };
exports.default = bitwise;
