// bitwise/bits/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.xor = exports.xnor = exports.toString = exports.toBoolean = exports.reduceXor = exports.reduceXnor = exports.reduceOr = exports.reduceNor = exports.reduceNand = exports.reduceAnd = exports.or = exports.not = exports.nor = exports.nand = exports.circularShiftRight = exports.circularShiftLeft = exports.and = void 0;
var and_1 = __importDefault(require("./and"));
exports.and = and_1.default;
var circular_shift_left_1 = __importDefault(require("./circular-shift-left"));
exports.circularShiftLeft = circular_shift_left_1.default;
var circular_shift_right_1 = __importDefault(require("./circular-shift-right"));
exports.circularShiftRight = circular_shift_right_1.default;
var nand_1 = __importDefault(require("./nand"));
exports.nand = nand_1.default;
var nor_1 = __importDefault(require("./nor"));
exports.nor = nor_1.default;
var not_1 = __importDefault(require("./not"));
exports.not = not_1.default;
var or_1 = __importDefault(require("./or"));
exports.or = or_1.default;
var reduce_and_1 = __importDefault(require("./reduce-and"));
exports.reduceAnd = reduce_and_1.default;
var reduce_nand_1 = __importDefault(require("./reduce-nand"));
exports.reduceNand = reduce_nand_1.default;
var reduce_nor_1 = __importDefault(require("./reduce-nor"));
exports.reduceNor = reduce_nor_1.default;
var reduce_or_1 = __importDefault(require("./reduce-or"));
exports.reduceOr = reduce_or_1.default;
var reduce_xnor_1 = __importDefault(require("./reduce-xnor"));
exports.reduceXnor = reduce_xnor_1.default;
var reduce_xor_1 = __importDefault(require("./reduce-xor"));
exports.reduceXor = reduce_xor_1.default;
var to_boolean_1 = __importDefault(require("./to-boolean"));
exports.toBoolean = to_boolean_1.default;
var to_string_1 = __importDefault(require("./to-string"));
exports.toString = to_string_1.default;
var xnor_1 = __importDefault(require("./xnor"));
exports.xnor = xnor_1.default;
var xor_1 = __importDefault(require("./xor"));
exports.xor = xor_1.default;
exports.default = {
    and: and_1.default,
    circularShiftLeft: circular_shift_left_1.default,
    circularShiftRight: circular_shift_right_1.default,
    nand: nand_1.default,
    nor: nor_1.default,
    not: not_1.default,
    or: or_1.default,
    reduceAnd: reduce_and_1.default,
    reduceNand: reduce_nand_1.default,
    reduceNor: reduce_nor_1.default,
    reduceOr: reduce_or_1.default,
    reduceXnor: reduce_xnor_1.default,
    reduceXor: reduce_xor_1.default,
    toBoolean: to_boolean_1.default,
    toString: to_string_1.default,
    xnor: xnor_1.default,
    xor: xor_1.default,
};
