// bitwise/buffer/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.xor = exports.xnor = exports.readUInt = exports.readInt = exports.read = exports.or = exports.not = exports.nor = exports.nand = exports.modify = exports.create = exports.and = void 0;
var and_1 = __importDefault(require("./and"));
exports.and = and_1.default;
var create_1 = __importDefault(require("./create"));
exports.create = create_1.default;
var modify_1 = __importDefault(require("./modify"));
exports.modify = modify_1.default;
var nand_1 = __importDefault(require("./nand"));
exports.nand = nand_1.default;
var nor_1 = __importDefault(require("./nor"));
exports.nor = nor_1.default;
var not_1 = __importDefault(require("./not"));
exports.not = not_1.default;
var or_1 = __importDefault(require("./or"));
exports.or = or_1.default;
var read_1 = __importDefault(require("./read"));
exports.read = read_1.default;
var read_int_1 = __importDefault(require("./read-int"));
exports.readInt = read_int_1.default;
var read_u_int_1 = __importDefault(require("./read-u-int"));
exports.readUInt = read_u_int_1.default;
var xnor_1 = __importDefault(require("./xnor"));
exports.xnor = xnor_1.default;
var xor_1 = __importDefault(require("./xor"));
exports.xor = xor_1.default;
exports.default = {
    and: and_1.default,
    create: create_1.default,
    modify: modify_1.default,
    nand: nand_1.default,
    nor: nor_1.default,
    not: not_1.default,
    or: or_1.default,
    read: read_1.default,
    readInt: read_int_1.default,
    readUInt: read_u_int_1.default,
    xnor: xnor_1.default,
    xor: xor_1.default,
};
